current/cfhp_all/collection.json

```diff
@@ -0,0 +1,1327 @@
+        "time": "2150"
+      }
+    },
+    {
+      "rel": "root",
+      "href": "../catalog.json",
+      "type": "application/json",
+      "title": "CoCliCo STAC Catalog"
+    },
+    {
+      "rel": "parent",
+      "href": "../catalog.json",
+      "type": "application/json",
+      "title": "CoCliCo STAC Catalog"
+    }
+  ],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+  ],
+  "item_assets": {
+    "data": {
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "Coastal Hazard Flood Projections",
+      "roles": [
+        "data"
+      ],
+      "description": "Coastal Flooding projections for Europe",
+      "xarray:storage_options": {
+        "token": "google_default"
+      }
+    }
+  },
+  "deltares:units": "Meters (m)",
+  "deltares:plot_type": "line",
+  "deltares:min": 0,
+  "deltares:max": 3,
+  "title": "Coastal Hazard Flood Projections",
+  "extent": {
+    "spatial": {
+      "bbox": [
+        [
+          1547750.0,
+          941625.0,
+          5415925.0,
+          6525950.0
+        ]
+      ]
+    },
+    "temporal": {
+      "interval": [
+        [
+          "2022-02-22T00:00:00Z",
+          "2022-02-22T00:00:00Z"
+        ]
+      ]
+    }
+  },
+  "license": "CC-BY-4.0",
+  "keywords": [
+    "Coast",
+    "Flood Maps",
+    "Coastal Hazard",
+    "Coastal Hazards",
+    "Flood Risk",
+    "Flood Projections",
+    "Coastal Hazard Flood Projections",
+    "Europe",
+    "EuropeanCoCliCo",
+    "Deltares",
+    "Cloud Optimized GeoTIFF",
+    "Natural Hazards",
+    "Full-Track"
+  ],
+  "providers": [
+    {
+      "name": "Deltares",
+      "roles": [
+        "processor",
+        "host"
+      ],
+      "url": "https://deltares.nl"
+    },
+    {
+      "name": "Universidad de Cantabria, Instituto de Hidr\u00e1ulica Ambiental de la Universidad de Cantabria 'IHCantabria'",
+      "roles": [
+        "producer"
+      ],
+      "url": "www.ihcantabria.com"
+    }
+  ],
+  "summaries": {
+    "defense level": [
+      "HIGH_DEFENDED_MAPS",
+      "LOW_DEFENDED_MAPS",
+      "UNDEFENDED_MAPS"
+    ],
+    "rp": [
+      "static",
+      "1",
+      "100",
+      "1000"
+    ],
+    "scenarios": [
+      "None",
+      "SSP126",
+      "SSP245",
+      "SSP585",
+      "High_End"
+    ],
+    "time": [
+      "2010",
+      "2100",
+      "2050",
+      "2030",
+      "2150"
+    ]
+  },
+  "assets": {
+    "thumbnail": {
+      "href": "https://storage.googleapis.com/dgds-data-public/coclico/assets/thumbnails/cfhp_all.png",
+      "type": "image/png",
+      "title": "Thumbnail"
+    },
+    "geoserver_link": {
+      "href": "https://coclico.avi.deltares.nl/geoserver/gwc/service/wmts?REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&LAYER=cba:GCF_open_CBA_country_all_EPSG4326&STYLE=&TILEMATRIX=EPSG:900913:{z}&TILEMATRIXSET=EPSG:900913&FORMAT=application/vnd.mapbox-vector-tile&TILECOL={x}&TILEROW={y}",
```

Comment:
This is not correct, need a link to the newly created cfhp_all_stats parquet file